packages/core/src/locales/zh-CN.yml

```
@@ -0,0 +1,7 @@
+wordle:
+  core:
+    wordle: Wordle
+    text-mode: |
+      Wordle
```

```suggestion
      {name}
```

Others games are not called `Wordle`, non hard-coding ici.